src/match_taxa_gbif_backbone.Rmd

```diff
@@ -0,0 +1,471 @@
+---
+title: "Match taxa with GBIF Taxonomy Backbone"
+author:
+  - Damiano Oldoni
+  - Lien Reyserhove
+  - Peter Desmet
+date: "`r Sys.Date()`"
+output:
+  html_document:
+    toc: true
+    toc_depth: 3
+    toc_float: true
+    number_sections: true
+    theme: yeti
+    df_print: paged
+knit: (function(input_file, encoding) { rmarkdown::render(input_file, encoding = encoding, output_file = paste0("../docs/",sub(".Rmd", ".html", basename(input_file))))})
+---
+
+# Setup
+
+## Load libraries
+
+```{r load_libs}
+library(odbc)       # To work with database
+library(tidyverse)  # To do data science
+library(here)       # To work with paths
+library(inborutils) # To match taxa (rgbif called in background)
+```
+
+## Load access informations
+
+Retrieve access informations from configuration file:
+
+```{r get_access_infos}
+ibge_bim <- config::get("ibge_bim")
+```
+
+# Read taxa
+
+Read taxa from `./data/interim/taxa.tsv`:
+
+```{r read_taxa_from_tsv_file}
+taxa <- read_tsv(here("data", "interim", "taxa.tsv"),
+                 col_types = cols(
+                   .default = col_character(),
+                   id = col_double(),
+                   parentid = col_double(),
+                   media = col_logical(),
+                   speciesbeparentid = col_double(),
+                   bruenvi_created = col_date(format = "%Y-%m-%d"),
+                   bruenvi_created = col_date(format = "%Y-%m-%d"),
+                   kingdom_id = col_double()
+                   )
+)
+```
+
+# Match taxa to GBIF Taxonomy Backbone
+
```

Comment:
I would add a short summary of the steps to undertake here, i.e.:
1. First match acceptedname + authorship + kingdom. 
2.  For non-exact matches: further match on acceptedname + kingdom
3. etc.
And shortly specify why. This makes the structure of the coming Rmd easier to follow.

Reply:
Yes, you are definitely right!
Here I added it: https://github.com/inbo/ibge-bim-species/pull/10/commits/3c223810fee196199919e3440a8cc8952d24c7f0